README.md

```diff
@@ -10,78 +10,64 @@
 
 </div>
 
-# ***Shifu***
-
 <div align="center">
 
 <img width="200px" src="./img/shifu-logo.svg"></img>
 
-</div>
-
-***Shifu*** is the next-generation Kubernetes-native open source IoT development framework, which makes developing an IoT app as simple as developing a Web app. The high extensibility of ***Shifu*** allows it to integrate almost all devices and protocols. Once a device is integrated, ***Shifu*** generates a digital twin of the device as a microservice in the system and opens up device data and capabilities in the form of APIs. In this way, traditional IoT application development is simplified to web development, which greatly improves the efficiency, quality and reusability of IoT application development.
-
-|Features<div style="width: 100pt">|  |
-|---|---|
-|⚡ Blazing-fast|From thermohydrometers using standard protocols to complex machinery using proprietary drivers, Shifu is capable of integrating all kinds of heterogeneous devices.|
-|🧩 Modularized|All devices and Apps integrated into Shifu are packaged as modules that can be loaded on demand.|
-|👨‍💻 Efficient|Once a device is integrated, Shifu automatically abstracts its capabilities into APIs, completely decoupling your App from the hardware, making IoT App development simple and efficient.|
-|🚀 Stable|Shifu is running in multiple production scenarios with 99.9999% stability, relieving you from the operational mess.|
-|🛡️ Safe |Designed by ex-UN cloud native security team. Shifu can easily enforce data encryption, network security and much more.|
-|🌐 Global Community|Benefiting from its Kubernetes-native architecture, Shifu can seamless integrate with the powerful cloud native software eco-system that allows developers around the world to help you with your problems.|
-
-## Guide
-
-### Install
-
-If you have started a Kubernetes cluster, use the command `kubectl apply` to install ***Shifu*** in your cluster:
-
-```sh
-cd shifu
-kubectl apply -f pkg/k8s/crd/install/shifu_install.yml
-```
 
-### Try it out
+Shifu is a Kubernetes-native IoT development framework that greatly improves the efficiency, quality and reusability of IoT application development.
 
-We present you [***Shifu*** Demo](https://shifu.run/disclaimer). 
 
-The Demo will show you how ***Shifu*** creates and manages digital twins of any physical devices.
-
-### Documentation
-
-For more information please visit our [documentation](https://shifu.run/docs/).
-
-## Design principle
-
-Our job is to make developers and operators happy. Which is why all our designs are developer-oriented:
-
-### 📡Easy to deploy
+|Feature|  |
+|---|---|
+|🔌Fast Device integration &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Compatible with almost all protocols and drivers.|
+|👨‍💻Efficient Application development|Shifu structually virtualizes each device and expose its capabilities in the form of APIs.|
```

Comment:
Layout issue in the second line Efficient Application development

also shall we put a space between the icon and the text? they looks too close